docs/source/commandline/commandline-all.rst

```diff
@@ -1121,7 +1121,7 @@ Core Sound Options
 
 **-audio_latency** *<value>*
 
-	This controls the amount of latency built into the audio streaming. By default MAME tries to keep the DirectSound audio buffer between 1/5 and 2/5 full. On some systems, this is pushing it too close to the edge, and you get poor sound sometimes. The latency parameter controls the lower threshold. The default is *1* (meaning lower=1/5 and upper=2/5). Set it to 2 (**-audio_latency 2**) to keep the sound buffer between 2/5 and 3/5 full. If you crank it up to 4, you can *definitely* notice audio lag.
+	This controls the amount of latency built into the audio streaming. By default MAME tries to keep the DirectSound audio buffer between 2/5 and 3/5 full. On some systems, this is pushing it too close to the edge, and you get poor sound sometimes. The latency parameter controls the lower threshold. The default is *2* (meaning lower=2/5 and upper=3/5). Set it to 1 (**-audio_latency 1**) to keep the sound buffer between 1/5 and 2/5 full. If you crank it up to 4, you can *definitely* notice audio lag.
```

Comment:
I'm not sure this description is even correct for DirectSound, but the interpretation of the value depends on the selected module.

For example, it controls the buffer size for CoreAudio:
```
    m_headroom = m_sample_bytes * (clamped_latency() * sample_rate() / 40);
    m_buffer_size = m_sample_bytes * std::max<uint32_t>(sample_rate() * (clamped_latency() + 3) / 40, 256U);
```

Reply:
Hi, I didn't write the original and I'm not familiar with the codebase to make that change, so I've reverted that part.